contracts/src/v0.8/automation/dev/v2_2/AutomationRegistry2_2.sol

```diff
@@ -300,17 +299,19 @@ contract AutomationRegistry2_2 is AutomationRegistryBase2_2, OCR2Abstract, Chain
     s_signersList = signers;
     s_transmittersList = transmitters;
 
+    // TODO bitmap
```

Comment:
will remove this comment.

To confirm, is this the place where we build the hotvars? if so, seems we are using some existing values to build the hotvar object, e.g. reentrancyGuard: s_hotVars.reentrancyGuard. want to make sure this is correct.

Reply:
Updated.